seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -4631,6 +4631,7 @@
         "assert(phoneticLookup(\"echo\") === 'Easy', 'message: <code>phoneticLookup(\"echo\")</code> should equal <code>\"Easy\"</code>');",
         "assert(phoneticLookup(\"foxtrot\") === 'Frank', 'message: <code>phoneticLookup(\"foxtrot\")</code> should equal <code>\"Frank\"</code>');",
         "assert(typeof phoneticLookup(\"\") === 'undefined', 'message: <code>phoneticLookup(\"\")</code> should equal <code>undefined</code>');",
+        "assert(code.match(/return\\sresult;/), 'message: You do not need to modify the <code>return</code> statement');",
```

Comment:
I like this change. I wonder about the message, it doesn't seem to match the style of the other messages. Would 'You should not modify the \<code\>return\</code\> statement' fit better? That matches the style of the line below. I can confirm that this code works as expected.